tests/regression/lp-1812973/Makefile

```diff
@@ -0,0 +1,21 @@
+name=lp-1812973
+
+CFLAGS=-Wall -Werror
+
+.PHONY: all
+all: $(name)
+
+.PHONY: clean
+clean:
+	rm -f $(name)
+
+DESTDIR ?=
+
+.PHONY: install
+install: $(name)
+	install -m 755 -d $(DESTDIR)/usr/bin/
+	install -m 755 $(name) $(DESTDIR)/usr/bin/
+
+.PHONY: fmt
+fmt: $(name).c
+	clang-format -style='{BasedOnStyle: Google, IndentWidth: 4, ColumnLimit: 120}' -i $^
```

Comment:
Nitpick: perhaps we should move style definition to a central file (clang-format supports style def from external file), it's already used in cmd/Makefile.am.

Reply:
Great idea, done!